bridges/GithubPackagesBridge.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+class GithubPackagesBridge extends BridgeAbstract
+{
+    const NAME = 'Github Packages Bridge';
+    const MAINTAINER = 'rsd76';
+    const URI = 'https://github.com/';
+    const DESCRIPTION = 'Bridge for getting GitHub packages';
+    const CACHE_TIMEOUT = 15;
+
+    const PARAMETERS = [
+        'Github organization/repository' => [
```

Comment:
Just don't give this entry a name.
```suggestion
        [
```